File: week-02/day-01/draw_square.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %   %
# %   %
# %   %
# %   %
# %%%%%
#
# The square should have as many lines as the number was
a = int(input("Enter a number: "))

for i in range(a):
    print("%", end="")
print()
if a > 1:
    for j in range(a - 2):
        print("%", end="")
        for i in range(a - 2):
            print(" ", end="")
        print("%")
    for i in range(a):
        print("%", end="")

File: week-07/day-1/draw-square.js
'use strict';

var lineCount = 6;

// Write a program that draws a
// square like this:
//
//
// %%%%%
// %   %
// %   %
// %   %
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is

for(let i = 0; i < lineCount; i++){
    let line = '';
    if(i === 0 || i === lineCount - 1){
        for(let j = 0; j < lineCount; j++){
            line += '%';
        }
    } else {
        line += '%';
        for(let j = 0; j < lineCount - 2; j++){
            line += ' ';
        }
        line += '%';
    }
    console.log(line);
}